app/components/provider_interface/application_choice_header_component.rb

```diff
@@ -57,10 +63,20 @@ def offer_present?
       ApplicationStateChange::OFFERED_STATES.include?(application_choice.status.to_sym)
     end
 
+    def interviews_present?
+      return false unless FeatureFlag.active?(:interviews)
+
+      application_choice.interviewing? && application_choice.interviews.any?
```

Comment:
Does `interviews.any?` take into account the possibility of cancelled interviews?

Reply:
Good catch. I assumed that the gem works like by adding a default scope (like [paranoia](https://github.com/rubysherpas/paranoia) but that doesnt seem to be the case.

I can either update this check of implement a default scope on the interviews model to do that. I think the **former is probably the best idea** as we don't want to potential impact other areas where we might want to access deleted interviews e.g. support

Reply:
👍🏼